code/src/components/Overview.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { Wrapper, Actionbtn } from 'styles/GlobalStyles';
+import { tasks } from 'reducers/tasks';
+
+export const Overview = ({ incomplete, allTasks, dispatch }) => {
+  // Function for completing all tasks when 'Complete all' button is clicked
+  const completeAll = () => {
+    allTasks.forEach(() => {
+      dispatch(tasks.actions.completeAllItems())
+    })
+  };
+
+  // Function for deleting all tasks when 'Remove all' button is clicked
+  const deleteAll = () => {
+    const userConfirm = window.confirm('Are you sure you want to remove all tasks?')
+    if (userConfirm) {
```

Comment:
I like this window alert! it's user friendly to ask, and not remove everything so easily 👍